api/tests/unit/features/workflows/core/test_unit_workflows_models.py

```diff
@@ -107,6 +107,30 @@ def test_change_request_commit_not_scheduled(
     assert change_request_no_required_approvals.feature_states.first().live_from == now
 
 
+def test_change_request_commit_when_minimum_change_request_approvals_is_none(
```

Comment:
Wouldn't it be simpler to just test the `is_approved` method? I'm not sure we need to double up on the testing here?